apps/judicial-system/backend/src/app/modules/case/internalCase.service.ts

```diff
@@ -1099,6 +1106,32 @@ export class InternalCaseService {
     return { delivered }
   }
 
+  async deliverSignedCourtRecordToPolice(
+    theCase: Case,
+    user: TUser,
+  ): Promise<DeliverResponse> {
+    const delivered = await this.getSignedCourtRecordPdf(theCase)
+      .then((pdf) =>
+        this.deliverCaseToPoliceWithFiles(theCase, user, [
+          {
+            type: PoliceDocumentType.RVTB,
```

Comment:
I haven't removed this type from the original case delivery to police, so investigation cases will end up with 2 different court records - one signed and one unsigned.. We feel like that should be alright, but if anyone knows of a reason we shouldn't do that then let me know. 
The reason is that in cases where there is a delay in signing it at least the prosecutors get the unsigned copy straight away..